version/version.go

```diff
@@ -1,9 +1,9 @@
 package version
 
 const (
-	// CMTSemVer is used as the fallback version of CometBFT
-	// when not using git describe. It uses semantic versioning format.
-	CMTSemVer = "1.0.0-dev"
```

Comment:
Sorry, but I don't agree with this change. See my comment [here](https://github.com/cometbft/cometbft/issues/3868#issuecomment-2313201043).